docs/config.rst

```diff
@@ -1,11 +1,606 @@
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+There are four steps to this flow. The first three steps are themselves flows, and the last step is a task.
+
+All *non-negative numbers and decimals* are valid as step numbers in a flow.
+You can add steps before, in-between, or after existing flow steps.
+If you want to add a step at the beginning of the dev org flow, valid step numbers would include any number greater than or equal to zero and less than 1. Example values would include 0, 0.3, and 0.89334. All of these would cause the step to execute before step 1 in the ``dev_org`` flow.
+
+If you want to add a step *between* steps 2 and 3, then a step number greater than 2 or less than 3 can be used.
+
+If you want to add a step *after* all steps in the flow, then any step number greater than 4 can be used.
+
+You can also add an additional log line output during the execution of a flow.
+
+    Example: ``dev_org``
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            5:
+                task: log
+                    options:
+                        line: dev_org flow has completed
+
+
+Skip a Flow Step
+^^^^^^^^^^^^^^^^
+
+To skip a flow step, set the desired step number to a task or flow with the value of ``None``.
+
+    Example: To skip the fourth step of the ``dev_org`` flow, insert the following under the ``flows`` section of your ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: None
+
+.. note::
+    The value of ``task`` must be used when skipping a flow step that is a task. The value of ``flow`` must be used when skipping a flow step that corresponds to a flow.
+
+When CumulusCI detects a task with a value of ``None``, the task is skipped.
+
+.. image:: images/skipping_task.png
+
+
+Replace a Flow Step
+^^^^^^^^^^^^^^^^^^^
+
+To replace a flow step, simply name the task or flow you wish to run in place of the current step.
+
+    Example: To replace the default fourth step of the ``dev_org`` flow with a custom task that loads data into a dev environment, specify the custom task you want to use in that step.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: load_data_dev
+
+___
```

Comment:
Does anyone know how to insert a blank character so RST knows when to break away from the code block? I can't indent the next example like I normally would because the code-block right before it will recognize the indented text as part of the code-block.

Reply:
I think that because you're indenting line 331 (the line that follows this) it thinks it is still part of this code block.

Reply:
RST allows the indent to act as a nested paragraph, so I've been using that whenever there's an Example. Do we not depend on RST in the final version of the doc?

And I put that "____" as a placeholder (in lieu of possibly a blank character that still occupies space?) because otherwise code-block keeps trying to include the indented formatting.

Reply:
@justinvholt I think you can end the code-block directive before starting a block quote by putting `..` unindented on its own line. This is the rST syntax for an empty comment.

Reply:
@davisagli You are THE MAN. Bet your bottom dollar I'll be using this handy feature every chance I get.